pkg/operator/starter.go

```diff
@@ -449,6 +458,14 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		os.Getenv("OPERATOR_IMAGE"),
 	)
 
+	backupController := backupcontroller.NewBackupController(
+		AlivenessChecker,
+		operatorConfigClientv1Alpha1,
+		kubeClient,
+		controllerContext.EventRecorder,
+		os.Getenv("OPERATOR_IMAGE"),
```

Comment:
Nit:
So we're mapping the `OPERATOR_IMAGE` to `targetImagePullSpec` arg.
Looking around in other controllers it seems we use `OPERATOR_IMAGE => operatorImagePullSpec` and `IMAGE => targetImagePullSpec`(for the etcd image).

Perhaps we can change the arg name from `targetImagePullSpec` to `operatorImagePullSpec` so it's consistent with how other controllers use that arg?

Reply:
Also just curious, for the backup pod itself, does it matter which image we use?
Seems like the upgradebackupcontroller's backup pod used the etcd image to run the actual backup script (and the operator image to verify storage in the init container):

https://github.com/openshift/cluster-etcd-operator/blob/a662347dba24b5c10462bf2273ee7959c2d24f0f/pkg/operator/upgradebackupcontroller/upgradebackupcontroller.go#L291-L295


https://github.com/openshift/cluster-etcd-operator/blob/a662347dba24b5c10462bf2273ee7959c2d24f0f/pkg/operator/starter.go#L438-L449

And we're using the operator image for that instead. Seems like we download etcdctl in the backup script so it doesn't matter either way.

Reply:
Thanks for raising this, that's what caused me to change the image yesterday. 
Renamed all references in that controller to operatorImagePullSpec, I leave a note on the other PR too.

> Also just curious, for the backup pod itself, does it matter which image we use?

it shouldn't, all tools are either mounted or downloaded. For some reason it was crashing with the operand image yesterday, let me try this again. Will post after the cluster is built.

Reply:
`oc` is missing:

```
        message: |
          + /usr/local/bin/cluster-backup.sh /etc/kubernetes/cluster-backup/backup-testbackup-2023-07-19_095939
          /usr/local/bin/cluster-backup.sh: line 45: oc: command not found
          Could not find the status of the kube-apiserver. Check if the API server is running. Pass the --force flag to skip checks.

```

Reply:
maybe the mount path has changed in the meantime?
it's in `/usr/bin` now

Reply:
it was using --force, which skips the use of OC. I'll mimic the upgrade controller in terms of their images though, so we're at least consistent.